webpack/webpack.development.js

```diff
@@ -10,8 +10,7 @@ module.exports = merge(common, {
   plugins: [
     new CopyPlugin({
       patterns: Object.keys((new AwsSamPlugin({ vscodeDebug: false }).entry())).map((lambdaName) => ([
-        { from: './.env', to: `.aws-sam/build/${lambdaName}/`, noErrorOnMissing: true },
-        { from: './node_modules/@dvsa/cvs-type-definitions/json-schemas/', to: `.aws-sam/build/${lambdaName}/json-schemas`}
+        { from: './.env', to: `.aws-sam/build/${lambdaName}/`, noErrorOnMissing: true }
```

Comment:
Looks to me like we don't need to copy the `json-schemas` from `node-modules` if we add `@dvsa/cvs-type-definitions` as a dependency, not a dev-dependency as it it now. Which kind of makes sense since it's actual "code" used at runtime.

Seems to also work for the prod build.